src/main/java/com/aws/greengrass/clientdevices/auth/configuration/RuntimeConfiguration.java

```diff
@@ -71,6 +73,25 @@ public String getCaPassphrase() {
         return Coerce.toString(caPassphrase);
     }
 
+    /**
+     * Returns the value of the client keystore passphrase.
+     */
+    public String getClientPassphrase() {
```

Comment:
This is just to store the keystore passphrase on the runtime. Again similar to the CertificateStore